components/popup/readme.md

```diff
@@ -0,0 +1,25 @@
+# vwc-popup
+
+Popup are used to display a message or notification to the user and are displayed on top of all other web page elements.
+Popup's goal is to provide additional, helpful content.   
+To trigger the Popup, it should be paired with an anchor (e.g., a button).
+
+## Properties
+
+| Property      | Attribute     | Type                   | Default    |
+| ------------- | ------------- | ---------------------- | ---------- |
+| `anchor`      | `anchor`      | `Element \| undefined` |            |
+| `arrow`       | `arrow`       | `boolean \| undefined` | true       |
+| `corner`      | `corner`      | `Placement`            | "left"     |
```

Comment:
- what does `Placement` mean in regards to wording? (`TOP_START` etc')
- can we set the popup direction as well? like in `transform-origin`

Reply:
a discussion is required regarding the corner property. snackbar uses position. while menu uses corner

Reply:
@yinonov The placement of the popup accepts  | 'top' | 'top-start' | 'top-end' | 'right' | 'right-start' | 'right-end' | 'bottom' | 'bottom-start' | 'bottom-end' | 'left' | 'left-start' | 'left-end'

Reply:
Then that's what needs to be documented